### PR TITLE
ACCUMULO-4575 Fixed concurrent delete issue in FATE ops

### PR DESCRIPTION
**Pull request by keith-turner**

### CODE REVIEW

**Comment by keith-turner** [comment]
For #206 only delete table running concurrently with delete table was fixed.   This PR handles the case of other FATE ops running concurrently with delete table.

**Comment by keith-turner** [comment]
I ran ITs that exercised each fate op that I changed.

File: server/master/src/main/java/org/apache/accumulo/master/tableOps/TableRangeOp.java [hunk 1]
```diff
@@ -38,19 +37,24 @@
   private static final long serialVersionUID = 1L;
 
   private final String tableId;
+  private final String namespaceId;
```
Comment (ctubbsii) [inline server/master/src/main/java/org/apache/accumulo/master/tableOps/TableRangeOp.java]: I'm surprised if inserting a new field in-between other fields actually works. I would have expected the order of the existing fields being disrupted to have an effect on serialization. This should be tested.

Comment (keith-turner) [inline server/master/src/main/java/org/apache/accumulo/master/tableOps/TableRangeOp.java]: I did some experiments with this and order does not seem to matter.  I tried reordering existing fields and inserting new fields and it did not matter.  Also I found some information that suggest the fields name is stored by serialization.

Comment (ctubbsii) [inline server/master/src/main/java/org/apache/accumulo/master/tableOps/TableRangeOp.java]: The [Java Object Serialization Specification] seems to indicate that order doesn't matter, also. Very cool.

[1]: https://docs.oracle.com/javase/7/docs/platform/serialization/spec/version.html

**Comment by keith-turner** [comment]
merged 7b9a11a

**Pull request closed without merging**